syntax = "proto3";
package protodata;

message SerializationSettings {
  string file = 1;
}

message Stop{
  string name = 1;
  double lat = 2;
  double lng = 3;
}

message Bus {
  string name = 1;
  bool is_roundtrip = 2;
  repeated string stops = 3;
}


message TransportCatalogue {
  repeated Bus buses = 1;
  repeated Stop stops = 2;
  map<string, uint32> distances_between_stops = 3;
  string separator = 4;
}